python-backend/tests/document_manager/resources/test_document_manager_resource.py

```diff
@@ -29,14 +36,14 @@ def test_happy_path_file_upload(test_client, auth_headers, setup_info):
     }
     
     post_resp = test_client.post('/document-manager', data=data, headers=auth_headers['full_auth_header'])
+
+    assert post_resp.status_code == 200
+
     post_data = json.loads(post_resp.data.decode())
     filename = post_data['document_manager_guids'][0]+'.pdf'
     saved_file = open(f'{test_dir_path}/mines/{setup_info.get("mine_guid")}/tailings/{filename}') 
```

Comment:
Does this get cleaned up after the test completes?

Reply:
Yeah, there is a fixture at the top that runs for every function and yields in the middle. It checks for the folders before and after the test and removes them.